templates/ts/office-addin/infra/azure.bicep

```diff
@@ -1,25 +1,27 @@
 @maxLength(20)
 @minLength(4)
 param resourceBaseName string
-param staticWebAppSku string
+param storageSku string
```

Comment:
`templates/ts/office-json-addin` is copied from `office-addin`, should update too